cli: webinspector: Fix remark regarding enablement for iOS 18

gunesacar opened the pull request:
Replaces `Settings -> Safari -> ...` with `Settings -> Apps -> Safari -> ...` in the comments to avoid confusion.

https://developer.apple.com/documentation/safari-developer-tools/inspecting-ios#Enabling-inspecting-your-device-from-a-connected-Mac

<!---

Thanks for opening a PR on pymobiledevice3!

Please review the CONTRIBUTING.md first regarding the linters we use and how
do we enforce them.

-->

## For community

⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

gunesacar committed:
Fix Safari settings menu path in webinspector.py

Replace `Settings -> Safari -> ...` with `Settings -> Apps -> Safari -> ...` in the comments to avoid confusion

https://developer.apple.com/documentation/safari-developer-tools/inspecting-ios#Enabling-inspecting-your-device-from-a-connected-Mac

doronz88 reviewed (changes requested):
Thanks! But this is refactor was only made in iOS 18. Please include both paths

gunesacar committed:
Update webinspector.py

List iOS version-specific paths for enabling web inspector and remote automation

gunesacar commented:
Added version specific paths in the comments

doronz88 reviewed (approved):
Cool, Thanks!

doronz88 commented:
Looks like there are some redundant spaces that fail the linter. Please commit again after it passed the flake8 test. I recommend using `pre-commit install` so that you'll have the same linter locally before every commit

gunesacar committed:
Remove trailing spaces

gunesacar commented:
Oops. Fixed. Installed the pre-commit hooks and flake8 shows green now.

doronz88 commented:
Merged! Thanks!
